tests/conf/chesapeake_cvpr_7.yaml

```diff
@@ -12,7 +12,7 @@ experiment:
     num_classes: 7
     num_filters: 1
     ignore_zeros: False
-    imagenet_pretraining: False
+    imagenet_pretraining: True
```

Comment:
One of the test configs for BYOL uses True, the other uses False. This gives us more test coverage.